src/index.ts

```diff
@@ -81,7 +81,6 @@ export * from './plugins/deviceorientation';
 export * from './plugins/googlemaps';
 export * from './plugins/httpd';
 export * from './plugins/imagepicker';
-export * from './plugins/inappbrowser';
```

Comment:
This line exports everything in `inappbrowser.ts`
I highly doubt that replacing this with individual imports & exports would make a difference.